### ISSUE TITLE
Fetch (Post) throwing error when only returning HttpStatusCode.OK

### ISSUE THREAD

**Issue by jduppie** [Microsoft/react-native-windows#1022]
Hi

My fetch (post) function works in ios and android, but not in windows. I have to include something in the response body else I get an error.

**Response from API that causes error:**

```
HTTP/1.1 200 OK
Cache-Control: no-cache
Pragma: no-cache
Expires: -1
Server: Microsoft-IIS/10.0
X-AspNet-Version: 4.0.30319
X-Powered-By: ASP.NET
Date: Tue, 21 Feb 2017 01:42:44 GMT
Content-Length: 0
```

```

TypeError: Network request failed
    at XMLHttpRequest.xhr.onerror (C:\t\node_modules\whatwg-fetch\fetch.js:441)
    at XMLHttpRequest.dispatchEvent (C:\t\node_modules\event-target-shim\lib\event…:172)
    at XMLHttpRequest.setReadyState (C:\t\node_modules\react-native\Libraries\Netw…:542)
    at XMLHttpRequest.__didCompleteResponse (C:\t\node_modules\react-native\Libraries\Netw…:378)
    at C:\t\node_modules\react-native\Libraries\Netw…:482
    at RCTDeviceEventEmitter.emit (C:\t\node_modules\react-native\Libraries\Even…:181)
    at MessageQueue.__callFunction (C:\t\node_modules\react-native\Libraries\Batc…:236)
    at C:\t\node_modules\react-native\Libraries\Batc…:108
    at guard (C:\t\node_modules\react-native\Libraries\Batc…:46)
    at MessageQueue.callFunctionReturnFlushedQueue (C:\t\node_modules\react-native\Libraries\Batc…:107)
```

Returning something in the body doesn't cause it to break:

```
HTTP/1.1 200 OK
Cache-Control: no-cache
Pragma: no-cache
Content-Type: application/json; charset=utf-8
Expires: -1
Vary: Accept-Encoding
Server: Microsoft-IIS/10.0
X-AspNet-Version: 4.0.30319
X-Powered-By: ASP.NET
Date: Tue, 21 Feb 2017 01:44:59 GMT
Content-Length: 4

true

```

Any ideas?

**Comment by rozele** [comment]
definitely a bug in NetworkingModule.cs

**Comment by rozele** [comment]
Thanks for filing @jduppie.  Can you check if the PR #1068 fixes the issue?

**Comment by jduppie** [comment]
Hi Rozele

Checked the changes. It's working now.

Thanks a lot

**Comment by matthargett** [comment]
Fixed by #1068. Will cherry pick into 42-stable and 43-stable and publish new versions of those packages.
